properties.schema

```diff
@@ -511,140 +521,140 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer icon foreground colour"
+            "help": "Defines the Drawer Icon Btn inverse colour. Should be a colour that provides good contrast against the Drawer Icon Btn colour"
           },
           "drawer-icon-hover": {
             "title": "Drawer icon colour - hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer icon background colour when hovered over"
+            "help": "Defines the Drawer Icon Btn colour when hovered over."
           },
           "drawer-icon-inverted-hover": {
             "title": "Drawer icon colour - inverted hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer icon foreground colour when hovered over"
+            "help": "Defines the Drawer Icon Btn inverse colour when hovered over. Should be a colour that provides good contrast against the Drawer Icon Btn hover colour."
           },
           "drawer-item": {
             "title": "Drawer item colour",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer item background colour"
+            "help": "Sets the Drawer Item colour."
           },
           "drawer-item-inverted": {
             "title": "Drawer item colour - inverted",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer item foreground colour"
+            "help": "Defines the Drawer Item text / icon colour. Should be a colour that provides good contrast against the Drawer Item colour."
           },
           "drawer-item-hover": {
             "title": "Drawer item colour - hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer item background colour when hovered over"
+            "help": "Defines the Drawer Item colour when hovered over."
           },
           "drawer-item-inverted-hover": {
             "title": "Drawer item colour - inverted hover",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer item foreground colour when hovered over"
+            "help": "Defines the Drawer Item text / icon colour when hovered over. Should be a colour that provides good contrast against the Drawer Item hover colour."
           },
           "drawer-item-selected": {
             "title": "Drawer item colour - selected",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Drawer item background colour when selected"
+            "help": "Defines the Drawer Item colour when selected."
```

Comment:
I'm not sure this help text really adds anything as the title is sufficiently clear by itself.